_posts/2020-10-12-notes-from-community-call-1-october-2020.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Notes from community call - 1 October 2020"
+date: 2020-10-12
+author: Jan Ainali
+type: blogpost
+excerpt: Discussion about reusability, thics and public code in general
+categories:
+  - Community call
+---
+
+# 1 October 2020 Standard for Public Code community call
+
+## Update from the Foundation
+
+* We gave several conference talks:
+    * [State of the Source 2020: How Open Source in Public Administrations and Government is different](https://www.youtube.com/watch?v=R7B2809l6EU) by Boris
+    * [Comparing ecosystems: MariaDB and Public Code - MariaDB Server Fest 2020](https://www.youtube.com/watch?v=m_H4KZmAVtY) by Eric
+    * [Open Sourcing our Way to the SDGs](https://www.youtube.com/watch?v=FkeQzL5q5t4), a panel Ben was in.
+* OpenZaak had their first [Product steering meeting](https://lists.publiccode.net/hyperkitty/hyperkitty/list/openzaak-discuss@lists.publiccode.net/thread/FAHQNHUEJADVAGOR6DTMWXMXIGYE3BDP/) and [Technical steering meeting](https://lists.publiccode.net/hyperkitty/hyperkitty/list/openzaak-discuss@lists.publiccode.net/thread/5JGJ2WMIW4M6A5WTBIO5AKEO3HXC7V4I/).
+
+## Attendees
+
+* [Jan Ainali](https://publiccode.net/team/jan-ainali.html) (Chair)
+* [Elena Findley-de Regt](https://publiccode.net/team/elena-findley-de-regt.html)
+* [Eric Herman](https://publiccode.net/team/eric-herman.html)
+* [Laura Scheske](https://publiccode.net/team/laura-scheske.html)
+
+## Notes
+
+### Distinguish the requirements of the "Create reusable and portable code" criterion
+
+This issue arose during a Standard gap analysis where it was a bit unclear what the differences were. They are intended to have a different focus so a new wording was proposed to make it more clear. We all liked [this proposed change](https://github.com/publiccodenet/standard/pull/353), but will let it linger a few weeks more in case anyone else has a comment.
```

Comment:
```suggestion
This issue arose during a Standard gap analysis where it was a bit unclear what the differences were. They are intended to have a different focus so new wording was proposed to make it more clear. We all liked [this proposed change](https://github.com/publiccodenet/standard/pull/353), but will let it linger for a few more weeks in case anyone else has a comment.
```